sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -168,6 +168,22 @@ class RocksDBSuite extends SparkFunSuite {
     }
   }
 
+  test("RocksDBFileManager: create init dfs directory with unknown number of keys") {
+    val dfsRootDir = new File(Utils.createTempDir().getAbsolutePath + "/state/1/1")
```

Comment:
Let's simply use `withTempDir` like below test.

Reply:
`withTempDir` would create the directory, however, I don't want to create the directory. Otherwise, it would not report the file not found exception.

Reply:
Ah OK. Good point.